src/main/java/net/masterthought/cucumber/json/Element.java

```diff
@@ -15,7 +15,7 @@
 
     // Start: attributes from JSON file report
     private final String id = null;
-    private final String name = null;
+    private String name = null;
```

Comment:
name is immutable

Reply:
i change this to mutable because i use the Name in the report because I append " (${numRetries)" so that it shows up in the report. 

alternatively we could create a new element in the html/xslt or whatever.